lib/index.js

```diff
@@ -35,19 +33,33 @@ class Oppsy extends Events.EventEmitter {
 
         Hoek.assert(interval <= 2147483647, 'interval must be less than 2147483648');
         const host = Os.hostname();
-        this._interval = setInterval(() => {
 
-            Items.parallel.execute(this._tasks, (error, results) => {
+        this._interval = setInterval(async () => {
 
-                if (error) {
-                    this.emit('error', error);
-                }
-                else {
-                    results.host = host;
-                    this.emit('ops', results);
+            const tasks = [];
+            for (const taskName in this._tasks) {
+                tasks.push(this._tasks[taskName]());
+            }
+
+            try {
+                const results = await Promise.all(tasks);
+
+                const emit = {
+                    host
+                };
+
+                for (const taskName in this._tasks) {
```

Comment:
Especially since we end up doing it twice down here.

Reply:
See above